components/EditOutivity/index.js

```diff
@@ -0,0 +1,156 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+export default function EditOutivityForm({
+  onEditOutivity,
+  outivity,
+  setIsEdit,
+}) {
+  const [editedOutivity, setEditedOutivity] = useState({
+    id: outivity.id,
+    title: outivity.title,
+    area: outivity.area,
+    country: outivity.country,
+    image: outivity.image,
+    description: outivity.description,
+  });
+
+  const saveEdits = (event) => {
+    event.preventDefault();
+    onEditOutivity(editedOutivity);
+    setIsEdit(false);
+  };
+
+  return (
+    <StyledNewOutivitiesForm onSubmit={saveEdits}>
+      <h1>Edit Outivity</h1>
+      <StyledNewOutivitiesFormFields>
+        <StyledNewOutivitiesFormField>
+          <label htmlFor="outivityImage">Image</label>
+
+          <StyledNewOutivitiesFormInput
+            type="text"
+            name="outivityImage"
+            id="outivityImage"
+            value={editedOutivity.image}
+          />
+        </StyledNewOutivitiesFormField>
+        <StyledNewOutivitiesFormField>
+          <label htmlFor="outivityName">Outivity</label>
+          <StyledNewOutivitiesFormInput
+            type="text"
+            name="outivityName"
+            id="outivityName"
+            value={editedOutivity.title}
+          />
+        </StyledNewOutivitiesFormField>
+        <StyledNewOutivitiesFormField>
+          <label htmlFor="outivityArea">Area</label>
+          <StyledNewOutivitiesFormInput
+            type="text"
+            name="outivityArea"
+            id="outivityArea"
+            value={editedOutivity.area}
+          />
+        </StyledNewOutivitiesFormField>
+        <StyledNewOutivitiesFormField>
+          <label htmlFor="outivityCountry">Country</label>
+          <StyledNewOutivitiesFormInput
+            type="text"
+            name="outivityCountry"
+            id="outivityCountry"
+            value={editedOutivity.country}
+          />
+        </StyledNewOutivitiesFormField>
+        <StyledNewOutivitiesFormField>
+          <label htmlFor="outivityDescription">Description</label>
+          <StyledNewOutivitiesFormTextarea
+            name="outivityDescription"
+            id="outivityDescription"
+            value={editedOutivity.description}
+          />
+        </StyledNewOutivitiesFormField>
+        <StyledNewOutivitiesFormSpan>
+          <StyledCancelButton type="button" onClick={() => setIsEdit(false)}>
+            cancel
+          </StyledCancelButton>
+          <StyledSaveButton type="button" onClick={saveEdits}>
```

Comment:
And the save button has to be ``type="submit"``. It then don't need an onClick anymore.